test/functional/pages/ColetaEditPage.groovy

```diff
@@ -3,15 +3,20 @@ import geb.Page
 
 class ColetaEditPage extends Page{
 
-    def titulo = "Edit Coleta"
-    static url = "/ResS/coleta/edit/1"
+    def titulo = "Coleta List"
+    static url = "http://localhost:8070/ResS/coleta/edit/1"
```

Comment:
isso não deve ser feito. O teste que depende disso só vai funcionar se o domínio for localhost e se o deselvovedor usar a porta 8070. Portanto, esse teste depende da sua plataforma de desenvolvimento, o que não pode ocorrer.